src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java

```diff
@@ -347,6 +363,11 @@ public static void addEt3DataToRespondent(CaseData caseData) {
     }
 
     private static RespondentSumType addEt3Data(CaseData caseData, RespondentSumType respondent) {
+        if (YES.equals(respondent.getResponseReceived())
```

Comment:
Do we not need to null check these here too?

Reply:
As discussed privately, anything that only needs to be compared with YES.equals doesn't need null checking